linux_os/guide/system/accounts/accounts-pam/password_quality/password_quality_pwquality/accounts_password_pam_retry/oval/shared.xml

```diff
@@ -47,13 +59,31 @@
   {{{ object_pwquality_retry( path="/etc/pam.d/password-auth", test_ref="password_pam_pwquality_retry_password_auth") }}}
   {{%- endif %}}
 
-  <ind:textfilecontent54_state comment="upper bound of password_pam_retry" id="state_password_pam_retry_upper_bound" version="1">
-    <ind:subexpression datatype="int" operation="less than or equal" var_ref="var_password_pam_retry" />
+  <ind:textfilecontent54_state id="state_password_pam_retry_upper_bound" version="1"
+                               comment="upper bound of password_pam_retry">
+    <ind:subexpression datatype="int" operation="less than or equal"
+                       var_ref="var_password_pam_retry"/>
   </ind:textfilecontent54_state>
 
-  <ind:textfilecontent54_state comment="lower bound of password_pam_retry" id="state_password_pam_retry_lower_bound" version="1">
+  <ind:textfilecontent54_state id="state_password_pam_retry_lower_bound" version="1"
+                               comment="lower bound of password_pam_retry">
     <ind:subexpression datatype="int" operation="greater than">0</ind:subexpression>
   </ind:textfilecontent54_state>
 
-  <external_variable comment="External variable for pam_pwquality retry" datatype="int" id="var_password_pam_retry" version="1" />
+  <external_variable datatype="int" id="var_password_pam_retry" version="1"
+                     comment="External variable for pam_pwquality retry"/>
```

Comment:
Please, avoid doing changes like indentation and line breaks on lines not related with actual fixes and enhancements. This increases the amount of lines one has to review WRT to a change.

Next time separate the beautification changes in a separate commit.